modules/rubiconBidAdapter.js

```diff
@@ -368,26 +280,77 @@ function RubiconAdapter() {
           return memo;
         }, {'rpfl_elemid': bidRequest.placementCode});
 
-      try {
-        bidmanager.addBidResponse(bidRequest.placementCode, bid);
-      } catch (err) {
-        utils.logError('Error from addBidResponse', null, err);
-      }
-    });
-    // Run the Emily user sync
+      bids.push(bid);
+
+      return bids;
+    }, []);
+
     hasUserSyncFired = syncEmily(hasUserSyncFired);
+
+    return bids;
   }
+};
+
+function _adCpmSort(adA, adB) {
+  return (adB.cpm || 0.0) - (adA.cpm || 0.0);
+}
+
+function _getScreenResolution() {
+  return [window.screen.width, window.screen.height].join('x');
+}
 
-  function _adCpmSort(adA, adB) {
-    return (adB.cpm || 0.0) - (adA.cpm || 0.0);
+function _getDigiTrustQueryParams() {
+  function getDigiTrustId() {
+    let digiTrustUser = window.DigiTrust && ($$PREBID_GLOBAL$$.getConfig('digiTrustId') || window.DigiTrust.getUser({member: 'T9QSFKPDN9'}));
```

Comment:
This surfaces a serious lacking in the base class.

Ideally, I'd like to purge `$$PREBID_GLOBAL$$` from the 1.0 adapters (see #1508, #1510). I have a plan to handle it (`registerBidder((config) => spec)`)... but I don't think it's fair to ask you to make those changes, since they're with my code.

Apparently, my changes landed in `master` while i was at lunch today... so I'll spin up a new branch where we can hack away on this.

Reply:
I chatted with @mkendall07 about this yesterday. I'm going to be focused on server-side stuff for the foreseeable future, so I won't actually have any time to work on it :/.

I opened up #1587 with the quick changes I'd make to the base adapter. I don't really care if you look those over, or incorporate them into this PR.

I doubt anyone else will want to actually implement `registerBidder((config) => spec)`. Instead, I'd suggest doing `import config` and using it directly. Although it means you're stuck with singleton configs in prebid, it's still worthwhile to purge `$$PREBID_GLOBAL$$`  from the code, for the reasons mentioned in #1508